docs/Measuring_Resonances.md

```diff
@@ -164,6 +187,62 @@ probe_points:
 It is advised to start with 1 probe point, in the middle of the print bed,
 slightly above it.
 
+#### Configure ADXL345 With Pi Pico
+
+##### Flash the Pico Firmware
+
+On your Raspberry Pi, compile the firmware for the Pico.
+
+```
+cd ~/klipper
+make clean
+make menuconfig
+```
+![Pico menuconfig](img/klipper_pico_menuconfig.png)
+
+Now, while holding down the `BOOTSEL` button on the Pico, connect the Pico to
+the Raspberry Pi via USB. Compile and flash the firmware.
```

Comment:
I'm not sure, maybe @KevinOConnor can comment but I believe a 
`sudo apt-install pkg-config`
is needed

Reply:
Fine for me. I always use vanilla Linux distributions and I had to install it manually.